x-pack/plugins/enterprise_search/public/applications/shared/flash_messages/flash_messages.tsx

```diff
@@ -7,21 +7,27 @@
 
 import React, { Fragment } from 'react';
 
-import { useValues } from 'kea';
+import { useValues, useActions } from 'kea';
 
-import { EuiCallOut, EuiCallOutProps, EuiSpacer } from '@elastic/eui';
+import { EuiCallOut, EuiCallOutProps, EuiSpacer, EuiGlobalToastList } from '@elastic/eui';
 
-import { FLASH_MESSAGE_TYPES } from './constants';
+import { FLASH_MESSAGE_TYPES, DEFAULT_TOAST_TIMEOUT } from './constants';
 import { FlashMessagesLogic } from './flash_messages_logic';
 
 export const FlashMessages: React.FC = ({ children }) => {
-  const { messages } = useValues(FlashMessagesLogic);
+  return (
```

Comment:
This could be simplfied a bit:

```JavaScript
export const FlashMessages: React.FC = ({ children }) => (
  <>
    <Callouts>{children}</Callouts>
    <Toasts />
  </>
);
```

Reply:
I'm on the fence about implicit vs explicit component returns 🤔 The only reason I kinda just like leaving the explicit return is because it doesn't create a ton of whitespace diff changes if you ever add any kind of logic before the return (e.g. a useEffect, a boolean, whatever). Typically most of our components do have some sort of logic or Kea value/action so I've been just leaving the extra `return` 'just in case' the component ever gets more complex in the future.

After having typed that all out I now realize I don't actually care all that much lol so I'm down for whatever 🤣

Reply:
https://github.com/elastic/kibana/pull/95981/commits/af471e7115cb0a7362f5aa0fbe2220fe976f73bd